Makefile

```diff
@@ -6,7 +6,7 @@ CC	:= cc -std=c11
 CXX	:= c++
 CXX_VERSION := -std=c++14
 CFLAGS	:= -Wall -Wno-format-security -O2 -DVERSION=$(VERSION)
-LDFLAGS	:= -lpthread
+LDFLAGS	:= -lpthread -lncurses -ltinfo
```

Comment:
this is needed since the bootstrapped tools all depend on util/ and util/ now needed them. We could maybe make "term.h" its own library if we wanted to not have this here